ion/src/main/java/com/fasterxml/jackson/dataformat/ion/IonParser.java

```diff
@@ -35,14 +36,52 @@
 public class IonParser
     extends ParserMinimalBase
 {
+    /**
+     * Enumeration that defines all togglable features for Ion parsers.
+     */
+    public enum Feature implements FormatFeature // in 2.9
```

Comment:
Change to 2.12 -- also, it is not 100% required to add this since there are no features yet -- I am ok either way, but just mention it.

Reply:
Done, nice catch 👍